docs/modules/ROOT/pages/presets.adoc

```diff
@@ -0,0 +1,33 @@
+:cairo-version: https://crates.io/crates/cairo-lang-compiler/2.3.1[cairo 2.3.1]
+:scarb: https://docs.swmansion.com/scarb[scarb]
+
+= Presets
+
+List of "ready-to-deploy" presets available and their corresponding class hashes.
+
+:account: xref:/api/account.adoc#Account[Account.cairo]
+:erc20: xref:/api/erc20.adoc#ERC20[ERC20.cairo]
+:erc721: xref:/api/erc721.adoc#ERC721[ERC721.cairo]
+:cairo-and-sierra: https://docs.starknet.io/documentation/architecture_and_concepts/Smart_Contracts/cairo-and-sierra/[Cairo and Sierra]
+
+NOTE: Class hashes were computed using {cairo-version}.
```

Comment:
should we document our process to update the class hashes and the cairo version?

Reply:
Maybe adding a mention in the CONTRIBUTING guide?

Reply:
I kind of lean toward including it in the presets doc because it could also be helpful for users. With the `NOTE`, it's implied that the hash may change based on the version, but I don't think it hurts to be more explicit. In the `NOTE`, perhaps we can add something like: "Amendments to the code and updates to the Cairo compiler may alter the class hash outputs. The following class hashes were computed using..."

Reply:
If we don't like this^, +1 to throwing a mention in CONTRIBUTING. I'm not sure where else we could put it